bench/tasks/m-03-failing-tests/brief.md

```
@@ -0,0 +1,6 @@
+The tests in `test/` describe the correct behaviour of `mergeIntervals`; the
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)